File: repos_estudiantes_ALVAREZ_primera_entrega/ARUQUIPA_VARGAS_ANA_GLORIA/POLIMORFISMO/Ejercicio2Py_java/main.py
from Videojuegos import VideoJuego
if __name__ == "__main__":
    juego1 = VideoJuego("Mobile Legends", "Android", 10)
    juego2 = VideoJuego("Wild Rift","ps5") 

    print("\nJuego 1")
    juego1.mostrarInfo()

    print("\nJuego 2")
    juego2.mostrarInfo()

    print("\nAgregando un jugador a Juego 1:")
    juego1.agregaJugadores()

    print("\nAgregando jugadores a Juego 2:")
    juego2.agregarJugadoresDesdeTeclado()

    print("\nDatos actualizados")
    juego1.mostrarInfo()
    print()
    juego2.mostrarInfo()

File: repos_estudiantes_ALVAREZ_primera_entrega/ARUQUIPA_VARGAS_ANA_GLORIA/POLIMORFISMO/Ejercicio2Py_java/Main.java
package Ejercicio2Py_java;

public class Main {
    
    public static void main(String[] args) {
        Videojuego juego1 = new Videojuego("Mobile Legends", "Android", 10);
        Videojuego juego2 = new Videojuego("Wild Rift");
        System.out.println("\nJuego 1 ");
        juego1.mostrarInfo();

        System.out.println("\nJuego 2 ");
        juego2.mostrarInfo();

        System.out.println("\nAgregando un jugador a Juego 1:");
        juego1.agregarJugadores();

        System.out.println("\nAgregando jugadores a Juego 2:");
        juego2.agregarJugadoresDesdeTeclado();

        System.out.println("\nDatos actualizados");
        juego1.mostrarInfo();
        System.out.println();
        juego2.mostrarInfo();
    }
}
